web/elm/src/Dashboard/Dashboard.elm

```diff
@@ -254,7 +255,29 @@ handleCallback callback ( model, effects ) =
                 ( newModel, effects )
 
         AllJobsFetched (Err err) ->
-            ( { model | jobsError = Just Failed }, effects )
+            case err of
+                Http.BadStatus { status } ->
+                    if status.code == httpStatusNotImplemented then
+                        ( { model
+                            | jobsError = Just Disabled
+                            , jobs = Fetched Dict.empty
+                            , pipelines =
+                                model.pipelines
+                                    |> Maybe.map
+                                        (List.map
+                                            (\p ->
+                                                { p | jobsDisabled = True }
```

Comment:
This will clobber any "click to sync" overrides of `jobsDisabled`, which obviously isn't a concern right now, but it reminded me: should we stop hitting the ListAllJobs endpoint after the first 501? These calls aren't expensive on the ATC, but it probably makes sense to stop trying if it's easy, which I think it should be. You have a better sense of the `Dashboard.RequestBuffer` module, but if I'm not mistaken you could modify this line:

https://github.com/concourse/concourse/blob/3ac5db508adb9db262f52857d6d23ea2c92f6c7e/web/elm/src/Dashboard/Dashboard.elm#L154

to something like:

```elm
        (\model -> model.dragState /= NotDragging && model.jobsError /= Just Disabled)
```

Reply:
That feels reasonable. If I don't use that crazy over-engineered thing I made now, what is it even good for? 😝 942f5c74aff07f01dc6bf689be085b83946f9c69